subprojects/core-api/src/main/java/org/gradle/api/file/FileCopyDetails.java

```diff
@@ -60,11 +62,32 @@ public interface FileCopyDetails extends FileTreeElement, ContentFilterable {
 
     /**
      * Sets the Unix permissions of this file.
+     * <p>
+     * Use {@link #permissions(Action)} instead.
```

Comment:
❓ I think we shouldn't say "use X instead" for APIs that are not deprecated. Or we should deprecate them. We could use `@see` or something similar to point people to the new APIs perhaps. WDYT?

Reply:
Yeah, very last of the leftovers of moving deprecations to separate PR. Removing it.

Reply:
But, beyond that, I do like the idea of starting to point people to the new API, even while incubating.... "Consider using XXX instead?" Will give it a try...